### PR TITLE
Added starwars demo

### PR DESCRIPTION
**Pull request by evergreen700**
added starwars schemas and objects

### CODE REVIEW

**Comment by bwalsh** [comment]
@evergreen700 Nice work so far.   Quick turn around.   Can you move this from resources/ to test/fixtures ?

File: resources/swapi/character.yaml [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+- - gid
+- - label
+  - id
+links:
+- rel: homeworld
+  href: homeworld/{id}
+  templateRequired:
+  - id
+  targetSchema:
+    "$ref": homeworld
```
Comment (bwalsh) [inline resources/swapi/character.yaml:19]: There is no homeworld schema type.  should this be planet?

File: resources/swapi/character.yaml [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+- gid
+- id
+- label
+uniqueKeys:
+- - gid
+- - label
+  - id
+links:
+- rel: homeworld
+  href: homeworld/{id}
```
Comment (bwalsh) [inline resources/swapi/character.yaml:15]: should this be planet/{id}

File: resources/swapi/character.yaml [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+  templateRequired:
+  - id
+  targetSchema:
+    "$ref": homeworld
+  templatePointers: id
```
Comment (bwalsh) [inline resources/swapi/character.yaml:20]: (Pending resolution of how  has_one is represented in the data)
if "all references are arrays"
  either  /homeworld/-/id or /homeworld/0/id 
if "has_ones should be scalar"
  /homeworld/id

File: resources/swapi/character.yaml [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+  targetHints:
+    directionality: outbound
+    multiplicity: has_one
+    backref: character
```
Comment (bwalsh) [inline resources/swapi/character.yaml:24]: should this be plural ie characters

File: resources/swapi/character.yaml [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+  templateRequired:
+  - id
+  targetSchema:
+    "$ref": starships
+  templatePointers: id
+  targetHints:
+    directionality: outbound
+    multiplicity: has_one
+    backref: character
+properties:
```
Comment (bwalsh) [inline resources/swapi/character.yaml:58]: homeworld should appear as a property

**Review by bwalsh — CHANGES_REQUESTED** [review]

File: resources/swapi/character.yaml (hunk 2 above)
Comment (evergreen700) [reply resources/swapi/character.yaml:15]: If I understand it right, rel is the name of the relationship so I think it should be homeworld? However, href should be planet/{id}

File: resources/swapi/character.yaml (hunk 5 above)
Comment (evergreen700) [reply resources/swapi/character.yaml:58]: If it is a link, should it be treated as a property as well? I was understanding that the links and properties are grouped separately on the schema

Comment (evergreen700) [reply resources/swapi/character.yaml:58]: Should links be appearing as properties and links? I thought it was one or the other.

Comment (evergreen700) [reply resources/swapi/character.yaml]: Nevermind homeworld should be in the properties now

**Comment by bwalsh** [comment]
@evergreen700 Zach - I wrote a [test](https://github.com/bmeg/iceberg-schema-tools/blob/9099fdc547bf9eda56433ba1c67360683788f2ce/tests/unit/test_swapi.py) and made changes to the yaml schemas

**Comment by bwalsh** [comment]
Merging and rebasing becoming unwieldy,  moved to #17

**Comment by bwalsh** [comment]
Merging and rebasing becoming unwieldy, moved to https://github.com/bmeg/iceberg-schema-tools/pull/17

**Pull request closed without merging**